src/index.html

```diff
@@ -17,16 +17,26 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar"
+      data-qa="big"
+    >
```

Comment:
The `<form>` tag is incorrectly self-closing. In HTML, `<form>` elements should have both opening and closing tags to properly contain their child elements. Change `<form />` to `<form>` and ensure it has a corresponding `</form>` tag.